crates/ide-assists/src/handlers/into_to_qualified_from.rs

```diff
@@ -52,18 +52,24 @@ pub(crate) fn into_to_qualified_from(acc: &mut Assists, ctx: &AssistContext<'_>)
         == FamousDefs(sema, scope.krate()).core_convert_Into()?
     {
         let type_call = sema.type_of_expr(&method_call.clone().into())?;
-        let type_call_disp =
-            type_call.adjusted().display_source_code(db, scope.module().into(), true).ok()?;
+        let adjusted_tc = type_call.adjusted();
+
+        if adjusted_tc.is_unknown() && adjusted_tc.contains_unknown() {
+            return None;
+        }
+
+        let qualified_from = format!(
+            "<{}>::from({})",
+            adjusted_tc.display_source_code(db, scope.module().into(), true).ok()?,
+            receiver
+        );
```

Comment:
We can move everything but the `adjusted_tc.display_source_code(db, scope.module().into(), true).ok()?` line into the builder closure